dietpi/dietpi-software

```diff
@@ -2971,6 +2979,21 @@ _EOF_
 
 		fi
 
+		software_id=190 # Beets
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Installing
+
+			G_EXEC mkdir /etc/beets
```

Comment:
As this is not only used for the configuration, but also for the database, what do you think about using `/mnt/dietpi_userdata/beets` to align this with other cases?

FHS would be `/var/lib/beets/library.db`, but IMO it's nice to have everything in userdata, as long as it's not somehow preset by the APT package.

Reply:
The preset is actually `~/.config/beets`, but the bashrc.d script overrides that.

Reply:
Jep, I thought that and makes sense when we expect a single music library on the system 👍.